spec/models/payment_history/company_payment.rb

```
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
```

Spec files should have `_spec.rb` suffix, otherwise RSpec may silently ignore them while running the whole spec suite